Makefile

```diff
@@ -11,7 +11,7 @@ VCS_VER = $(shell git describe --tags --contains --always)
 all: lint build backup generate-accounts tests clean
 
 build:
-	docker build -t $(NAME) . --build-arg VCS_VER=$(VCS_VER) --build-arg VCS_REF=$(VCS_REF)
+	docker build --force-rm -t $(NAME) . --build-arg VCS_VER=$(VCS_VER) --build-arg VCS_REF=$(VCS_REF)
```

Comment:
~~I'm not sure about this, doesn't this re-build the image from scratch each time? I'd rather re-use layers when I can, especially for small changes such as in scripts that take no time to build a new image from.~~

**EDIT:** `--rm` is default, which does the same behaviour for successful builds already. `--force-rm` ensures that applies to failed builds (_where the containers were retained to assist with debugging_).

So I guess I was mistaken, and this doesn't change caching of past image builds, it only ensures intermediate containers (an images layers from directives in the Dockerfile) are removed during failed builds too.. Was that your intention?

It sounds like you'd still get previous image builds piling up in storage, perhaps you wanted `--no-cache`? (_which I don't since I prefer build performance and a VM that I discard afterwards for cleanup_)

If you do want `--no-cache` perhaps have a separate `clean-build` command?

Reply:
Hmm, not sure. I just want failed builds to not pile up and force me to manually delete them. Let me think about it

Reply:
Reverting this to propose it in another PR